README.md

```diff
@@ -443,12 +453,20 @@ To learn how to set up your editor for Bats syntax highlighting, see [Syntax
 Highlighting](https://github.com/bats-core/bats-core/wiki/Syntax-Highlighting)
 on the wiki.
 
+## Contributing
+
+For now see the ``docs`` folder for project guides, work with us on the wiki
+or look at the other communication channels.
+
+## Contact
+
+- We are `#bats` on freenode;
+- Or leave a message on [gitter].
```

Comment:
You're missing the Gitter URL

Reply:
Also, I can't find the `#bats` channel on freenode. Does it exist yet?

Reply:
This is from original docs, I only moved it. Perhaps better point to (and update) wiki contact page.

Reply:
Also the gitter URL works, the def is (was already) at the bottom.
See live version https://github.com/bvberkum/bats-core/tree/forks/bvb-docs

Reply:
Ah, didn't realize that. No, keeping it in the README is probably the best IMO.